app/src/main/java/com/f0x1d/logfox/extensions/FragmentExtensions.kt

```diff
@@ -1,5 +1,9 @@
 package com.f0x1d.logfox.extensions
 
 import androidx.fragment.app.Fragment
+import com.f0x1d.logfox.R
 
-fun Fragment.showAreYouSureDialog(okPressed: () -> Unit) = requireActivity().showAreYouSureDialog(okPressed)
\ No newline at end of file
+fun Fragment.showAreYouSureDialog(
+    title: Int,
+    message: Int = R.string.are_you_sure,
+    okPressed: () -> Unit) = requireActivity().showAreYouSureDialog(title, message, okPressed)
```

Comment:
`) = requireActivity().showAreYouSureDialog(title, message, okPressed)` should also start from new line :)